scripts/build-deb

```diff
@@ -62,8 +62,3 @@ if [ "${DIST_VERSION}" = 'n/a' ]; then
 	echo "Failed to get release codename"
 	exit 1
 fi
-
-# Only create an archive if env variable is specified
-if [ -n "${CREATE_ARCHIVE}" ]; then
-	tar -C /build -cvzf "/archive/${DIST_ID}-${DIST_VERSION}-${ARCH}.tar.gz" "${DIST_ID}/${DIST_VERSION}/${ARCH}"
-fi
```

Comment:
Note to self; instead of tarring here, perhaps we can use `docker build --output type=tar...` (although that doesn't support compressing, but I guess all artefacts are already compressed, so from that perspective, it should be fine)